src/main.rs

```diff
@@ -1,3 +1,33 @@
-fn main() {
-    println!("Hello, world!");
+use stand_control_bot::bot::{handlers::build_handler, BotContext, BotState};
+use std::{path::Path, sync::Arc};
+use teloxide::{dispatching::dialogue::InMemStorage, prelude::*};
+
+#[tokio::main]
+async fn main() {
+    {
+        let env_file = Path::new(".env");
+        if env_file.exists() {
+            dotenv::from_filename(".env").unwrap();
+        }
+    }
+    pretty_env_logger::formatted_timed_builder()
+        .parse_filters(&std::env::var("RUST_LOG").unwrap_or("DEBUG".to_string()))
+        .init();
+
+    log::info!("Starting bot...");
```

Comment:
I'm planning on setting up scaffolding for `tracing` in the next pr. I guess I should start with migrating `stand_control_bot` to it first